server/database/schema.sql

```diff
@@ -1,64 +1,53 @@
--- Vidéo
-CREATE TABLE video (
-  id INT UNSIGNED PRIMARY KEY AUTO_INCREMENT NOT NULL,
-  name VARCHAR(80) NOT NULL,
-  description TEXT,
-  date DATE NOT NULL,
-  is_premium BOOLEAN NOT NULL DEFAULT FALSE,
-  is_free BOOLEAN NOT NULL DEFAULT FALSE,
-  requires_account BOOLEAN NOT NULL DEFAULT FALSE
-);
-
 -- Catégorie
 CREATE TABLE category (
-  id INT UNSIGNED PRIMARY KEY AUTO_INCREMENT NOT NULL,
-  name VARCHAR(80) NOT NULL
+    id INT UNSIGNED PRIMARY KEY AUTO_INCREMENT NOT NULL,
+    name VARCHAR(80) NOT NULL
 );
 
-CREATE TABLE sub_category (
-  id INT UNSIGNED PRIMARY KEY AUTO_INCREMENT NOT NULL,
-  name VARCHAR(80) NOT NULL,
-  category_id INT UNSIGNED,
-  FOREIGN KEY(category_id) REFERENCES category(id)
+-- Vidéo
+CREATE TABLE video (
+    id INT UNSIGNED PRIMARY KEY AUTO_INCREMENT NOT NULL,
+    name VARCHAR(80) NOT NULL,
+    url VARCHAR(255) NOT NULL,
+    image VARCHAR(255) NOT NULL,
+    description TEXT,
+    date DATE NOT NULL,
+    is_premium BOOLEAN NOT NULL DEFAULT FALSE,
+    is_free BOOLEAN NOT NULL DEFAULT FALSE,
+    requires_account BOOLEAN NOT NULL DEFAULT FALSE,
+    category_id INT UNSIGNED,
+    FOREIGN KEY (category_id) REFERENCES category (id)
```

Comment:
Unsure if SQL likes the space between `category` and `(id)`
Ditto for the following foreign keys